Sources/EmbraceCrash/EmbraceCrashReporter.swift

```diff
@@ -55,16 +55,20 @@ public final class EmbraceCrashReporter: NSObject, CrashReporter {
             return
         }
 
-        if ksCrash.userInfo == nil {
-            ksCrash.userInfo = [:]
+        var crashInfo: [AnyHashable: Any] = [:]
+
+        if ksCrash.userInfo != nil {
+            crashInfo = ksCrash.userInfo
         }
 
-        ksCrash.userInfo[UserInfoKey.sdkVersion] = self.sdkVersion ?? NSNull()
-        ksCrash.userInfo[UserInfoKey.sessionId] = self.currentSessionId ?? NSNull()
+        crashInfo[UserInfoKey.sdkVersion] = self.sdkVersion ?? NSNull()
+        crashInfo[UserInfoKey.sessionId] = self.currentSessionId ?? NSNull()
```

Comment:
Can we move these to after the extraInfo values are set? Would want these keys to not be overridable

Reply:
This is the way it did behave in the past. Maybe, we can split `sdkVersion` and `currentSessionId` to their own methods instead

Reply:
No I just mean I don't want anyone to be able to call `Embrace.client.appendCrashInfo` and override the sdkVersion or sessionId.

This is still possible if you reach directly into KSCrash but hoping that we could prevent accidents.

Reply:
Oh I see; ok, I can do it to prevent this. Plus, i'll add a test to check that possibility